config/admissions.uiowa.edu/field.field.fragment.stat.field_stat_title.yml

```diff
@@ -0,0 +1,32 @@
+uuid: c5baa8fc-eb5a-49bb-9bc0-c4758b16807a
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.fragment.field_stat_title
+    - fragments.fragment_type.stat
+  module:
+    - heading
+id: fragment.stat.field_stat_title
+field_name: field_stat_title
+entity_type: fragment
+bundle: stat
+label: Title
```

Comment:
One thing I had done in #2759 was to call this "Figure" instead of "Title," since I find the latter to be confusing and my guess is that it is also confusing to most editors that are not experienced with the component already.

Reply:
Is changing the label on this and the existing Stat block enough or do you want me to do machine names for these fragment fields (prefix and suffix have `title` in the machine name)?